NeoMathEngine/src/GPU/CUDA/Rowwise/CudaRowwiseMobileNetV2.h

```diff
@@ -68,9 +68,12 @@ class CCudaRowwiseMobileNetV2 : public ICudaRowwiseImpl, public CRowwiseOperatio
 	const CConstFloatHandle downFreeTerm;
 	const int outputChannels;
 	const bool residual;
-	CBlobDesc inputDesc;
-	CBlobDesc outputDesc;
-	std::unique_ptr<CChannelwiseConvolutionDesc> chDesc;
+	// Reshape
+	CBlobDesc inputDesc{};
+	CBlobDesc outputDesc{};
+	std::unique_ptr<CChannelwiseConvolutionDesc> chDesc{};
+
+	friend class CCudaMathEngine;
```

Comment:
Why `friend` here? `IMathEngine::MobileNetV2Block` is `public`

Reply:
Now the `CudaMathEngine::MobileNetV2Block` used the `private` fields of `CCudaRowwiseChConvWith1x1` and `CCudaRowwiseMobileNetV2` structs, but these structs are not nested types as in `CpuMathEngine` class.